code/game/objects/items/weapons/powerfist.dm

```diff
@@ -31,7 +31,7 @@
 
 /obj/item/melee/powerfist/attackby(obj/item/W, mob/user, params)
 	if(istype(W, /obj/item/tank/internals))
-		if(!iscarbon(loc))
+		if(!iscarbon(loc) && user.get_active_hand())
 			to_chat(user, "<span class='warning'>You have to hold the powerfist in your hand!</span>")
```

Comment:
```suggestion
			to_chat(user, "<span class='warning'>You have to hold [scr] in your hand!</span>")
```

Reply:
src*

Reply:
i swear to god i TRIPLE CHECKED and still got it wrong